build/configs/artik05x/artik05x_download.sh

```diff
@@ -145,11 +145,18 @@ compute_ocd_commands()
 
 download()
 {
-        parts_default=`grep -A 2 'config FLASH_PART_NAME' ${PARTITION_KCONFIG} | sed -n 's/\tdefault "\(.*\)".*/\1/p'`
-        parts2=${CONFIG_FLASH_PART_NAME:=${parts_default}}
-        parts=`echo $parts2 | sed "s/,/ /g"`
+	parts=$1;
 
-        # Make Openocd commands for parts
+	if [[ -n $parts ]] && [[ "$parts" != "all" ]] && [[ "$parts" != "ALL" ]]
+	then
+		echo "##Download $parts"
```

Comment:
And if you want to add the provision before hand, you can do one thing.
add below lines 
`
echo "feature is yet to be implemented"
exit 0
`
So at least it will not result in an error

Reply:
It isn't before hand. As said, how did you test it? Paste the logs here.

Reply:
It is solved now
thanks